src/contexts/__tests__/current-product.test.tsx

```diff
@@ -48,11 +48,32 @@ describe('CurrentProductContext', () => {
 	})
 
 	test('useCurrentProduct throws an error if not used within CurrentProductProvider', () => {
-		const { result } = renderHook(() => useCurrentProduct())
+		const spy = jest.spyOn(console, 'error').mockImplementation(() => void 0)
 
-		expect(result.error.message).toBe(
```

Comment:
`result.error` no longer exists, so I had to add some code to capture the error and silence the console.